scripts/build.py

```diff
@@ -609,6 +611,80 @@ def build_files(source, target, file_hashes):
             print e
 
 
+def build_HTML_files(source, target, file_hashes):
```

Comment:
Typically we go with build_html_files() (no caps)

Reply:
Function removed!